aggregator/handler/config.go

```diff
@@ -171,6 +175,41 @@ func (c flushHandlerConfiguration) Validate() error {
 	return nil
 }
 
+type trafficControlConfiguration struct {
```

Comment:
nit: move this to be after `staticBackendConfiguration` struct definition to roughly match the ordering in which the types appear.